LAMPAPI/DeleteContact.php

```diff
@@ -1,7 +1,7 @@
 <?php
 	$inData = getRequestInfo();
 	
-	$id = $inData["id"];
+	$UID = $inData["UID"];
```

Comment:
this is the variable name that was being used in the JS. Used window.SessionStorage to hold variable value. Even after redirecting to the Update Contacts page.